src/verifyConditions.ts

```diff
@@ -1,20 +1,33 @@
-import {Context} from "semantic-release";
-import {Config} from "./Config";
+import SemanticReleaseError from "@semantic-release/error";
+import { getDiscordVars } from "./lib/getDiscordVars";
 
-export function verifyConditions(config: Config, context: Context) {
-    const {env, options, logger} = context;
-    const errors: string[] = [];
-    if (!options.plugins.find(p => p === '@semantic-release/release-notes-generator'
-        || p?.[0] === '@semantic-release/release-notes-generator')) {
-        logger.error("The plugin @semantic-release/release-notes-generator was not found.");
-        errors.push('missing-release-notes-generator');
-    }
-    if (!env.DISCORD_WEBHOOK && !config.webhook) {
-        logger.error("No webhook found in config or env environment variable \"DISCORD_WEBHOOK\".", 'no-webhook');
-        errors.push('missing-hook');
-    }
+export function verifyConditions(getDiscordVars, context) {
```

Comment:
`getDiscordVars` is redefined, it was imported line 2, change this variable name.